includes/class-admin.php

```diff
@@ -70,6 +73,59 @@ public static function manage_comments_custom_column( $column, $comment_id ) {
 		echo esc_html( get_webmention_comment_type_string( $comment_id ) );
 	}
 
+	/**
+	 * Add bulk option to bulk comment handler
+	 */
+	public static function bulk_comment_actions( $bulk_actions ) {
+		$bulk_actions['refresh_webmention'] = __( 'Refresh Webmention', 'webmention' );
+		return $bulk_actions;
+	}
+
+	/**
+	 * Add bulk action handler to comments
+	 *
+	 */
+	public static function bulk_comment_action_handler( $redirect_to, $doaction, $comment_ids ) {
```

Comment:
this is still a lot of code for refreshing a comment... maybe we should move this to the class-comment.php file, so that we could call it from several places...

Reply:
it is also hard to check if it works or not, maybe we could also add a test for that?

Reply:
I did add a refresh timestamp but agree the function to refresh should be broken off